esmvaltool/recipes/hydrology/recipe_hype.yml

```diff
@@ -0,0 +1,39 @@
+# ESMValTool
+# recipe_pcrglobwb.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim reanalyses files for use in the Hype hydrological model.
+
+
+  authors: ['pelupessy_inti']
+  projects: ['ewatercycle']
+  references: ['acknow_project']
+
+
+datasets:
+  - {dataset: ERA-Interim, project: OBS, tier: 3, type: reanaly, version: 1}
+
+preprocessors:
+  preprocessor:
+    extract_shape:
+      shapefile: HYPE_Shapefiles/Meuse_HYPE/Meuse_HYPE.shp
```

Comment:
where is this stored other than your local repo? Are you going to add this to the package? If so, how big is it?

Reply:
> where is this stored other than your local repo? Are you going to add this to the package? If so, how big is it?

We will probably make a separate repo for all auxiliary files we need for the hydrological preprocessing in the [eWaterCycle organization](https://github.com/eWaterCycle)

Reply:
should we change the name of the file here?

Reply:
The filename is just an example, so this should be fine.